lib/rex/post/meterpreter/core_ids.rb

```diff
@@ -47,6 +47,8 @@ module Meterpreter
 COMMAND_ID_CORE_TRANSPORT_SLEEP          = EXTENSION_ID_CORE + 33
 COMMAND_ID_CORE_PIVOT_SESSION_NEW        = EXTENSION_ID_CORE + 34
 
+COMMAND_ID_CORE_SET_TERM_SIZE            = EXTENSION_ID_CORE + 337
```

Comment:
This is just a placeholder for now, I'm not even sure this command should exist in core or if it should be in stdapi?